gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py

```diff
@@ -96,17 +98,19 @@ def test_update_pool_stats(self):
             context = test_data.Context()
             agent.update_pool_stats('pool_id', 'stats', context)
 
+            data = {'info': {'service_type': 'loadbalancer',
```

Comment:
This data is specific to this api but lb_test_data.py has resources data which is used across all apis. So kept it here.